features/support/testing_dsl.rb

```diff
@@ -112,6 +112,33 @@ def clone(fixture_name)
     class EmptyProject < Project
     end
 
+    class CondaProject < Project
+      CONDA_ROOT = '/root/miniconda3'
```

Comment:
Same question as before about hardcoded paths. Not a problem, just curious if this will always be the case or just most of the time.

Reply:
It will always be the same since it's driven by the way we install conda in the container.